Documentation/kubernetes-upgrade.md

```diff
@@ -0,0 +1,68 @@
+# Upgrading Kubernetes
+
+This document will describe upgrading your cluster's master and worker nodes. For general information on Kubernetes cluster management, upgrades (including more advanced topics such as major API version upgrades) see the [Kubernetes upstream documentation](http://kubernetes.io/v1.0/docs/admin/cluster-management.html).
+
+## Upgrading the Kubelet
+
+The Kubelet binary ships with CoreOS releases. As the host OS is updated, the Kubelet will be upgraded as well.
+
+If you wish to run a custom version of the kubelet binary, the kubelet service file (`/etc/systemd/system/kubelet.service`) should be modified to use the custom binary path on each node (master & worker).
+
+## Upgrading Master Nodes
+
+Master nodes will consist of the following Kubernetes components:
+
+* kubelet
```

Comment:
You should just drop this from the list given the disclaimer above about how the kubelet is managed.